app/controllers/home_controller.rb

```diff
@@ -0,0 +1,9 @@
+class HomeController < ApplicationController
+	def index
+		@products = Product.where(retired: false)
+		@product_picker = Product.where(retired: false).where.not(id: nil).sample(3)
```

Comment:
Yeaahh.. hmm I think I meant to put if the image was not nil, in case a user didn't add an image to their product.